crates/optimism/node/src/node.rs

```diff
@@ -552,17 +548,12 @@ impl<NetworkT> OpAddOnsBuilder<NetworkT> {
 
 impl<NetworkT> OpAddOnsBuilder<NetworkT> {
     /// Builds an instance of [`OpAddOns`].
-    pub fn build<N>(
-        self,
-    ) -> OpAddOns<
-        N,
-        OpEthApiBuilder<NetworkT>,
-        OpEngineValidatorBuilder,
-        OpEngineApiBuilder<OpEngineValidatorBuilder>,
-    >
+    pub fn build<N, EV, EB>(self) -> OpAddOns<N, OpEthApiBuilder<NetworkT>, EV, EB>
     where
         N: FullNodeComponents<Types: NodeTypes>,
         OpEthApiBuilder<NetworkT>: EthApiBuilder<N>,
+        EV: Default,
+        EB: Default,
```

Comment:
I think this is fine, but this makes me think we could also move this to the OpAddOnsBuilder itself